device-registry/src/main/scala/org/genivi/sota/device_registry/db/GroupInfoRepository.scala

```diff
@@ -88,15 +70,11 @@ object GroupInfoRepository extends SlickJsonHelper {
       .result
       .failIfNotSingle(Errors.MissingGroupInfo)
 
-  def exists(id: Uuid, namespace: Namespace)(implicit ec: ExecutionContext): DBIO[GroupInfo] =
+  def exists(id: Uuid, namespace: Namespace)(implicit ec: ExecutionContext): DBIO[Group] =
     groupInfos
       .filter(d => d.namespace === namespace && d.id === id)
       .result
       .headOption
-      .flatMap(_.fold[DBIO[GroupInfo]](DBIO.failed(Errors.MissingGroupInfo))(DBIO.successful))
-
-  def discardedAttrs(groupId: Uuid)(implicit ec: ExecutionContext): DBIO[GroupInfoType] =
-    findById(groupId)
-      .map(_.discardedAttrs)
+      .flatMap(_.fold[DBIO[Group]](DBIO.failed(Errors.MissingGroupInfo))(DBIO.successful))
```

Comment:
`failIfNone(Errors.MissingGroupInfo)` instead of this line.. Maybe rename the Errors makes sense.